src/languageProvider/locales/en_US.json

```diff
@@ -224,7 +224,7 @@
   "str.time": "Time",
   "str.to": "To",
   "str.token": "Token",
-  "str.transaction": "TRANSACTIONS",
+  "str.transaction": "My Transactions",
```

Comment:
remove this string and make a new string called `str.myTransactions`. We already have existing translations in Korean and Chinese for `str.transaction` so if any part of the string changes in English we should be removing any old translations and update with new translations.